src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -202,31 +202,30 @@ internal void AttachToOwner(View owner)
         /// <param name="heightMeasureSpec">Vertical space requirements as imposed by the parent.</param>
         /// <since_tizen> 6 </since_tizen>
         public void Measure(MeasureSpecification widthMeasureSpec, MeasureSpecification heightMeasureSpec)
+        {
+            OnMeasure(widthMeasureSpec, heightMeasureSpec);
+            OnMeasureIndependentChildren(widthMeasureSpec, heightMeasureSpec);
+            flags = flags | LayoutFlags.LayoutRequired;
+            flags &= ~LayoutFlags.ForceLayout;
+
+            oldWidthMeasureSpec = widthMeasureSpec;
+            oldHeightMeasureSpec = heightMeasureSpec;
+        }
+
+        internal bool NeedsLayout(float widthSize, float heightSize, MeasureSpecification.ModeType widthMode, MeasureSpecification.ModeType heightMode)
         {
             // Check if relayouting is required.
-            bool specChanged = (widthMeasureSpec.Size != oldWidthMeasureSpec.Size) ||
-                (heightMeasureSpec.Size != oldHeightMeasureSpec.Size) ||
-                (widthMeasureSpec.Mode != oldWidthMeasureSpec.Mode) ||
-                (heightMeasureSpec.Mode != oldHeightMeasureSpec.Mode);
+            bool specChanged = (widthSize != oldWidthMeasureSpec.Size.AsDecimal()) || (widthMode != oldWidthMeasureSpec.Mode) ||
+                               (heightSize != oldHeightMeasureSpec.Size.AsDecimal()) || (heightMode != oldHeightMeasureSpec.Mode);
 
-            bool isSpecExactly = (widthMeasureSpec.Mode == MeasureSpecification.ModeType.Exactly) &&
-                (heightMeasureSpec.Mode == MeasureSpecification.ModeType.Exactly);
+            bool isSpecExactly = (widthMode == MeasureSpecification.ModeType.Exactly) &&
+                                 (heightMode == MeasureSpecification.ModeType.Exactly);
 
-            bool matchesSpecSize = (MeasuredWidth.Size == widthMeasureSpec.Size) &&
-                (MeasuredHeight.Size == heightMeasureSpec.Size);
+            bool matchesSpecSize = (MeasuredWidth.Size.AsDecimal() == widthSize) && (MeasuredHeight.Size.AsDecimal() == heightSize);
 
             bool needsLayout = specChanged && (!isSpecExactly || !matchesSpecSize);
-            needsLayout = needsLayout || ((flags & LayoutFlags.ForceLayout) == LayoutFlags.ForceLayout);
 
-            if (needsLayout)
-            {
-                OnMeasure(widthMeasureSpec, heightMeasureSpec);
-                OnMeasureIndependentChildren(widthMeasureSpec, heightMeasureSpec);
-                flags = flags | LayoutFlags.LayoutRequired;
-                flags &= ~LayoutFlags.ForceLayout;
-            }
-            oldWidthMeasureSpec = widthMeasureSpec;
-            oldHeightMeasureSpec = heightMeasureSpec;
+            return LayoutRequested || needsLayout;
```

Comment:
How about comparing separately?

```
if (LayoutRequested) // line 217
{
    return true;
}
```

```
return needsLayout; //last line of NeedsLayout
```

Reply:
Thank you. I've fixed patch.

I agree with the problem you mentioned. I've modified the logic so that it doesn't change.
When the Width / Height is changed, the SetFrame will also change by measure. 

I think the problem is position. I decided that the change of position is not a factor in the layouting.
However, logic changes may affect the operation, so I will reflect it after more testing.